### ISSUE TITLE
Autocomplete on switch for object of type T crashes Visual Studio 2019

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#37009]
When using the autocomplete feature visual studio breaks (reproducibly) when trying to enter a type for a switch statement inside a type T function:

```csharp
private void TypeDelegate<T>(T value)
{
    switch (value)
    {
       case 
       default:
           break;
    }
}

```
When the cursor is after the case, it start looking for types. VS 2019 crashes and restarts reproducibly.

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/590659/autocomplete-on-switch-for-object-of-type-t-crashe.html
VSTS ticketId: 903464_
_These are the original issue comments:_

Visual Studio Feedback System on 6/2/2019, 07:30 PM (32 days ago): <p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

Visual Studio Feedback System on 7/5/2019, 11:36 AM (24 min ago): <markdown><p>This issue is currently being investigated. Our team will get back to you if either more information is needed, a workaround is available, or the issue is resolved.</p>
</markdown>
_These are the original issue solutions:_
(no solutions)

**Comment by sharwell** [comment]
Stack trace for the crash:

```
Event Time (UTC): 5/31/2019 10:40:30 AM
Event ID: 1025
Data: Application: devenv.exe
Framework Version: v4.0.30319
Description: The application requested process termination through System.Environment.FailFast(string message).
Message: System.NullReferenceException: Object reference not set to an instance of an object.

   at Microsoft.CodeAnalysis.CSharp.ConversionsBase.ClassifyImplicitBuiltInConversionSlow(TypeSymbol source, TypeSymbol destination, HashSet`1& useSiteDiagnostics)

   at Microsoft.CodeAnalysis.CSharp.ConversionsBase.ClassifyBuiltInConversion(TypeSymbol source, TypeSymbol destination, HashSet`1& useSiteDiagnostics)

   at Microsoft.CodeAnalysis.CSharp.Binder.ExpressionOfTypeMatchesPatternType(Conversions conversions, TypeSymbol expressionType, TypeSymbol patternType, HashSet`1& useSiteDiagnostics, Conversion& conversion, ConstantValue operandConstantValue, Boolean operandCouldBeNull)

   at Microsoft.CodeAnalysis.CSharp.Binder.ConvertPatternExpression(TypeSymbol inputType, CSharpSyntaxNode node, BoundExpression expression, ConstantValue& constantValue, Boolean hasErrors, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.ConvertCaseExpression(CSharpSyntaxNode node, BoundExpression caseExpression, Binder sectionBinder, ConstantValue& constantValueOpt, DiagnosticBag diagnostics, Boolean isGotoCaseExpr)

   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BuildSwitchLabels(SyntaxList`1 labelsSyntax, Binder sectionBinder, ArrayBuilder`1 labels, DiagnosticBag tempDiagnosticBag)

   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BuildLabels()

   at Microsoft.CodeAnalysis.CSharp.LocalScopeBinder.get_Labels()

   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.get_LabelsByNode()

   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BindSwitchSection(SwitchSectionSyntax node, Binder originalBinder, BoundSwitchLabel& defaultLabel, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BindSwitchSections(Binder originalBinder, DiagnosticBag diagnostics, BoundSwitchLabel& defaultLabel)

   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BindSwitchStatementCore(SwitchStatementSyntax node, Binder originalBinder, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.Binder.BindSwitchStatementCore(SwitchStatementSyntax node, Binder originalBinder, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.Binder.BindSwitchStatementCore(SwitchStatementSyntax node, Binder originalBinder, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.Binder.BindSwitchStatement(SwitchStatementSyntax node, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.Binder.BindStatement(StatementSyntax node, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.IncrementalBinder.BindStatement(StatementSyntax node, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.Bind(Binder binder, CSharpSyntaxNode node, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.MethodBodySemanticModel.Bind(Binder binder, CSharpSyntaxNode node, DiagnosticBag diagnostics)

   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.GetBoundNodes(CSharpSyntaxNode node)

   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.GetLowerBoundNode(CSharpSyntaxNode node)

   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.GetBoundNodes(CSharpSyntaxNode node, CSharpSyntaxNode& bindableNode, BoundNode& lowestBoundNode, BoundNode& highestBoundNode, BoundNode& boundParent)

   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.GetTypeInfoWorker(CSharpSyntaxNode node, CancellationToken cancellationToken)

   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetTypeInfo(ExpressionSyntax expression, CancellationToken cancellationToken)

   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetTypeInfoFromNode(SyntaxNode node, CancellationToken cancellationToken)

   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetTypeInfoCore(SyntaxNode node, CancellationToken cancellationToken)

   at Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.GetTypesSimple(SyntaxNode node)

   at Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.GetTypes_DoNotCallDirectly(SyntaxNode node, Boolean objectAsDefault)

   at Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypeInSwitchLabel(SwitchLabelSyntax switchLabel, Nullable`1 previousToken)

   at Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService.TypeInferrer.InferTypesWorker_DoNotCallDirectly(Int32 position)

   at Microsoft.CodeAnalysis.LanguageServices.TypeInferenceService.AbstractTypeInferenceService.InferTypes(SemanticModel semanticModel, Int32 position, String nameOpt, CancellationToken cancellationToken)

   at Microsoft.CodeAnalysis.Shared.Extensions.ITypeInferenceServiceExtensions.InferType(ITypeInferenceService typeInferenceService, SemanticModel semanticModel, Int32 position, Boolean objectAsDefault, CancellationToken cancellationToken)

   at Microsoft.CodeAnalysis.CSharp.Completion.Providers.EnumAndCompletionListTagCompletionProvider.<ProvideCompletionsAsync>d__1.MoveNext()
Stack:
   at System.Environment.FailFast(System.String, System.Exception)
   at Microsoft.CodeAnalysis.FailFast.OnFatalException(System.Exception)
   at Microsoft.CodeAnalysis.ErrorReporting.FatalError.Report(System.Exception, System.Action`1<System.Exception>)
   at Microsoft.CodeAnalysis.ErrorReporting.FatalError.ReportUnlessCanceled(System.Exception)
   at Microsoft.CodeAnalysis.CSharp.Completion.Providers.EnumAndCompletionListTagCompletionProvider+<ProvideCompletionsAsync>d__1.MoveNext()
   at Microsoft.CodeAnalysis.CSharp.ConversionsBase.ClassifyImplicitBuiltInConversionSlow(Microsoft.CodeAnalysis.CSharp.Symbols.TypeSymbol, Microsoft.CodeAnalysis.CSharp.Symbols.TypeSymbol, System.Collections.Generic.HashSet`1<Microsoft.CodeAnalysis.DiagnosticInfo> ByRef)
   at Microsoft.CodeAnalysis.CSharp.ConversionsBase.ClassifyBuiltInConversion(Microsoft.CodeAnalysis.CSharp.Symbols.TypeSymbol, Microsoft.CodeAnalysis.CSharp.Symbols.TypeSymbol, System.Collections.Generic.HashSet`1<Microsoft.CodeAnalysis.DiagnosticInfo> ByRef)
   at Microsoft.CodeAnalysis.CSharp.Binder.ExpressionOfTypeMatchesPatternType(Microsoft.CodeAnalysis.CSharp.Conversions, Microsoft.CodeAnalysis.CSharp.Symbols.TypeSymbol, Microsoft.CodeAnalysis.CSharp.Symbols.TypeSymbol, System.Collections.Generic.HashSet`1<Microsoft.CodeAnalysis.DiagnosticInfo> ByRef, Microsoft.CodeAnalysis.CSharp.Conversion ByRef, Microsoft.CodeAnalysis.ConstantValue, Boolean)
   at Microsoft.CodeAnalysis.CSharp.Binder.ConvertPatternExpression(Microsoft.CodeAnalysis.CSharp.Symbols.TypeSymbol, Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode, Microsoft.CodeAnalysis.CSharp.BoundExpression, Microsoft.CodeAnalysis.ConstantValue ByRef, Boolean, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.ConvertCaseExpression(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode, Microsoft.CodeAnalysis.CSharp.BoundExpression, Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.ConstantValue ByRef, Microsoft.CodeAnalysis.DiagnosticBag, Boolean)
   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BuildSwitchLabels(Microsoft.CodeAnalysis.SyntaxList`1<Microsoft.CodeAnalysis.CSharp.Syntax.SwitchLabelSyntax>, Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.PooledObjects.ArrayBuilder`1<Microsoft.CodeAnalysis.CSharp.Symbols.LabelSymbol>, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BuildLabels()
   at Microsoft.CodeAnalysis.CSharp.LocalScopeBinder.get_Labels()
   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.get_LabelsByNode()
   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BindSwitchSection(Microsoft.CodeAnalysis.CSharp.Syntax.SwitchSectionSyntax, Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.CSharp.BoundSwitchLabel ByRef, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BindSwitchSections(Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.DiagnosticBag, Microsoft.CodeAnalysis.CSharp.BoundSwitchLabel ByRef)
   at Microsoft.CodeAnalysis.CSharp.SwitchBinder.BindSwitchStatementCore(Microsoft.CodeAnalysis.CSharp.Syntax.SwitchStatementSyntax, Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.Binder.BindSwitchStatementCore(Microsoft.CodeAnalysis.CSharp.Syntax.SwitchStatementSyntax, Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.Binder.BindSwitchStatementCore(Microsoft.CodeAnalysis.CSharp.Syntax.SwitchStatementSyntax, Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.Binder.BindSwitchStatement(Microsoft.CodeAnalysis.CSharp.Syntax.SwitchStatementSyntax, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.Binder.BindStatement(Microsoft.CodeAnalysis.CSharp.Syntax.StatementSyntax, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel+IncrementalBinder.BindStatement(Microsoft.CodeAnalysis.CSharp.Syntax.StatementSyntax, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.Bind(Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.MethodBodySemanticModel.Bind(Microsoft.CodeAnalysis.CSharp.Binder, Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode, Microsoft.CodeAnalysis.DiagnosticBag)
   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.GetBoundNodes(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode)
   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.GetLowerBoundNode(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode)
   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.GetBoundNodes(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode, Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode ByRef, Microsoft.CodeAnalysis.CSharp.BoundNode ByRef, Microsoft.CodeAnalysis.CSharp.BoundNode ByRef, Microsoft.CodeAnalysis.CSharp.BoundNode ByRef)
   at Microsoft.CodeAnalysis.CSharp.MemberSemanticModel.GetTypeInfoWorker(Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetTypeInfo(Microsoft.CodeAnalysis.CSharp.Syntax.ExpressionSyntax, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetTypeInfoFromNode(Microsoft.CodeAnalysis.SyntaxNode, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetTypeInfoCore(Microsoft.CodeAnalysis.SyntaxNode, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService+TypeInferrer.GetTypesSimple(Microsoft.CodeAnalysis.SyntaxNode)
   at Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService+TypeInferrer.GetTypes_DoNotCallDirectly(Microsoft.CodeAnalysis.SyntaxNode, Boolean)
   at Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService+TypeInferrer.InferTypeInSwitchLabel(Microsoft.CodeAnalysis.CSharp.Syntax.SwitchLabelSyntax, System.Nullable`1<Microsoft.CodeAnalysis.SyntaxToken>)
   at Microsoft.CodeAnalysis.CSharp.CSharpTypeInferenceService+TypeInferrer.InferTypesWorker_DoNotCallDirectly(Int32)
   at Microsoft.CodeAnalysis.LanguageServices.TypeInferenceService.AbstractTypeInferenceService.InferTypes(Microsoft.CodeAnalysis.SemanticModel, Int32, System.String, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.Shared.Extensions.ITypeInferenceServiceExtensions.InferType(Microsoft.CodeAnalysis.LanguageServices.ITypeInferenceService, Microsoft.CodeAnalysis.SemanticModel, Int32, Boolean, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.CSharp.Completion.Providers.EnumAndCompletionListTagCompletionProvider+<ProvideCompletionsAsync>d__1.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[[Microsoft.CodeAnalysis.CSharp.Completion.Providers.EnumAndCompletionListTagCompletionProvider+<ProvideCompletionsAsync>d__1, Microsoft.CodeAnalysis.CSharp.Features, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35]](<ProvideCompletionsAsync>d__1 ByRef)
   at Microsoft.CodeAnalysis.CSharp.Completion.Providers.EnumAndCompletionListTagCompletionProvider.ProvideCompletionsAsync(Microsoft.CodeAnalysis.Completion.CompletionContext)
   at Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders+<GetContextAsync>d__33.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Start[[Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders+<GetContextAsync>d__33, Microsoft.CodeAnalysis.Features, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35]](<GetContextAsync>d__33 ByRef)
   at Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders.GetContextAsync(Microsoft.CodeAnalysis.Completion.CompletionProvider, Microsoft.CodeAnalysis.Document, Int32, Microsoft.CodeAnalysis.Completion.CompletionTrigger, Microsoft.CodeAnalysis.Options.OptionSet, System.Nullable`1<Microsoft.CodeAnalysis.Text.TextSpan>, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders+<ComputeNonEmptyCompletionContextsAsync>d__26.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[[System.Collections.Immutable.ImmutableArray`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]], System.Collections.Immutable, Version=1.2.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a]].Start[[Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders+<ComputeNonEmptyCompletionContextsAsync>d__26, Microsoft.CodeAnalysis.Features, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35]](<ComputeNonEmptyCompletionContextsAsync>d__26 ByRef)
   at Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders.ComputeNonEmptyCompletionContextsAsync(Microsoft.CodeAnalysis.Document, Int32, Microsoft.CodeAnalysis.Completion.CompletionTrigger, Microsoft.CodeAnalysis.Options.OptionSet, Microsoft.CodeAnalysis.Text.TextSpan, System.Collections.Immutable.ImmutableArray`1<Microsoft.CodeAnalysis.Completion.CompletionProvider>, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders+<GetCompletionsAsync>d__24.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Start[[Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders+<GetCompletionsAsync>d__24, Microsoft.CodeAnalysis.Features, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35]](<GetCompletionsAsync>d__24 ByRef)
   at Microsoft.CodeAnalysis.Completion.CompletionServiceWithProviders.GetCompletionsAsync(Microsoft.CodeAnalysis.Document, Int32, Microsoft.CodeAnalysis.Completion.CompletionTrigger, System.Collections.Immutable.ImmutableHashSet`1<System.String>, Microsoft.CodeAnalysis.Options.OptionSet, System.Threading.CancellationToken)
   at Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.AsyncCompletion.CompletionSource+<GetCompletionContextAsync>d__18.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Start[[Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.AsyncCompletion.CompletionSource+<GetCompletionContextAsync>d__18, Microsoft.CodeAnalysis.EditorFeatures, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35]](<GetCompletionContextAsync>d__18 ByRef)
   at Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.AsyncCompletion.CompletionSource.GetCompletionContextAsync(Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.IAsyncCompletionSession, Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Data.CompletionTrigger, Microsoft.VisualStudio.Text.SnapshotPoint, Microsoft.VisualStudio.Text.SnapshotSpan, System.Threading.CancellationToken)
   at Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.AsyncCompletionSession+<>c__DisplayClass88_1.<ConnectToCompletionSources>b__0()
   at Microsoft.VisualStudio.Text.Utilities.GuardedOperations+<CallExtensionPointAsync>d__34`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Start[[Microsoft.VisualStudio.Text.Utilities.GuardedOperations+<CallExtensionPointAsync>d__34`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]], Microsoft.VisualStudio.Platform.VSEditor, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a]](<CallExtensionPointAsync>d__34`1<System.__Canon> ByRef)
   at Microsoft.VisualStudio.Text.Utilities.GuardedOperations.CallExtensionPointAsync[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.Object, System.Func`1<System.Threading.Tasks.Task`1<System.__Canon>>, System.__Canon)
   at Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.AsyncCompletionSession+<ConnectToCompletionSources>d__88.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[[Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.CompletionSourceConnectionResult, Microsoft.VisualStudio.Platform.VSEditor, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a]].Start[[Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.AsyncCompletionSession+<ConnectToCompletionSources>d__88, Microsoft.VisualStudio.Platform.VSEditor, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a]](<ConnectToCompletionSources>d__88 ByRef)
   at Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.AsyncCompletionSession.ConnectToCompletionSources(Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Data.CompletionTrigger, Microsoft.VisualStudio.Text.SnapshotPoint, Microsoft.VisualStudio.Text.ITextSnapshot, System.Threading.CancellationToken)
   at Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.AsyncCompletionSession+<GetInitialModel>d__89.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Start[[Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.AsyncCompletionSession+<GetInitialModel>d__89, Microsoft.VisualStudio.Platform.VSEditor, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a]](<GetInitialModel>d__89 ByRef)
   at Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.AsyncCompletionSession.GetInitialModel(Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Data.CompletionTrigger, Microsoft.VisualStudio.Text.SnapshotPoint, Microsoft.VisualStudio.Text.ITextSnapshot, System.Threading.CancellationToken)
   at Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.AsyncCompletionSession+<>c__DisplayClass71_0.<Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.IAsyncCompletionSession.OpenOrUpdate>b__0(Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.CompletionModel, System.Threading.CancellationToken)
   at Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.ModelComputation`1+<>c__DisplayClass13_0+<<Enqueue>b__0>d[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].MoveNext()
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.InvokeMoveNext(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
   at System.Threading.Tasks.Task.ExecutionContextCallback(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(System.Threading.Tasks.Task ByRef)
   at System.Threading.Tasks.Task.ExecuteEntry(Boolean)
   at System.Threading.Tasks.TaskScheduler.TryExecuteTask(System.Threading.Tasks.Task)
   at Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.PrioritizedTaskScheduler.ThreadStart()
   at System.Threading.ThreadHelper.ThreadStart_Context(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Threading.ThreadHelper.ThreadStart()
```

**Comment by jcouv** [comment]
Verified this is fixed in 16.2p4. I think it was fixed by https://github.com/dotnet/roslyn/pull/35249
I'll go ahead and close. Thanks
